lua/cmake-tools/init.lua

```diff
@@ -133,13 +128,6 @@ function cmake.generate(opt, callback)
       if type(callback) == "function" then
         callback()
       end
-
-      if const.cmake_soft_link_compile_commands then
-        utils.softlink(
-          config.build_directory.filename .. "/compile_commands.json",
-          vim.loop.cwd() .. "/compile_commands.json"
-        )
-      end
     end,
```

Comment:
Why don't we have `cmake.configure_compile_commands()` here?

Reply:
You're right. I've added it.